app/data/test_language.json

```diff
@@ -11,18 +11,19 @@
         "id": "english-group",
         "title": "",
         "blocks": [{
-            "type": "Interstitial",
-            "id": "english-block",
-            "sections": [{
-                "title": "English language schema",
-                "id": "english-section",
-                "questions": []
-            }],
-            "title": ""
-        },
-        {
-            "type": "Summary",
-            "id": "summary"
-        }]
+                "type": "Interstitial",
+                "id": "english-block",
+                "sections": [{
+                    "title": "English language schema",
+                    "id": "english-section",
+                    "questions": []
+                }],
+                "title": ""
+            },
+            {
+                "type": "Summary",
+                "id": "summary"
+            }
+        ]
     }]
-}
+}
```

Comment:
It looks like some of the schemas have been reformatted, making it hard to see what the required changes are for this pull request. Please can any that have just been reformatted be reverted.